### PR TITLE
16733 undefined placeholder for federal

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* /bcgov/entity#16733

Added conditional statement for "FEDERAL"

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-569-p19rvkyr.web.app

File: src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue [hunk 1]
```diff
@@ -98,7 +98,6 @@ export default class ConsentContinuationOut extends Mixins(CountriesProvincesMix
     const foreignJusrisdictionCountry = this.filing.data?.consentContinuationOut?.country
     const country = this.getCountryName(this.filing.data?.consentContinuationOut?.country)
     const region = this.getRegionNameFromCode(this.filing.data?.consentContinuationOut?.region)
-
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue:101]: This blank line was intentional, to separate the variable declarations from the function code. Please revert it.

**Review by severinbeauvais — APPROVED** [review]

File: src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue]: TY

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**